recipe/meta.yaml

```diff
@@ -50,10 +50,15 @@ test:
     - hypothesis >=3.58
     - pytest >=6.0
     - pytest-xdist
+    - pytest-html
   imports:
     - pandas
+  source_files:
+    - pandas/tests
```

Comment:
We should _not_ copy the pandas/test; that should either:
* Already be in the package, making this part redundant; or
* Be packaged separately so we don't balloon the size of the end-user package.

Reply:
got it if its available to trigger will remove this copying.